Documentation/components/iptables/pcn-iptables.rst

```diff
@@ -46,20 +46,34 @@ Limitations
 ^^^^^^^^^^^
 
 - No support for multiple chains
-- No support for ``-i`` ``-o`` interfaces
 - No support for ``SNAT``, ``DNAT``, ``MASQUESRADE``
 - ``-S`` ``-L`` generate an output slightly different from iptables
 
 Install
 -------
 
+Prerequisites
+^^^^^^^^^^^^^
+
+pcn-iptables comes as a component of polycube framework.
+Refer to :doc:`polycube install guide<../../../installation>` for dependencies, kernel requirements and basic checkout and install guide.
+
+Install Steps
+^^^^^^^^^^^^^
+
 For ``pcn-iptables`` support you should enable ``ENABLE_PCN_IPTABLES`` flag in CMakeFile.
```

Comment:
Does this mean that we must compile Polycube with a special flag?
If yes, I would rewrite this as follows:

To compile ``pcn-iptables``, you should enable `the `ENABLE_PCN_IPTABLES`` flag in the polycube CMakeFile; in fact, the set of eBPF services that are required by pcn-iptables to operate are not compiled with the by default:

Reply:
Just to be clear, the default config is:
`ENABLE_SERVICE_IPTABLES`=`ON`: this flag compiles and install the libpcn-iptables.so service (similar to others services like bridge, router, ...)
`ENABLE_PCN_IPTABLES` =`OFF`: this flag compile the customized version of iptables, and install the pcn-iptables-init pcn-iptables alias and pcn-iptables-clean scripts on your system.

Reply:
It was not clear to me, and what I understood was that libpcn-iptables.so was not created by default. I would suggest to change the text in order to make it more clear. Thanks!

Reply:
Agree. I've added a more detailed description about flags.